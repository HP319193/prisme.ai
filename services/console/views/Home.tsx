import { NextPage } from 'next';
import { useTranslation } from 'react-i18next';
import { Button } from '@prisme.ai/design-system';
import Link from 'next/link';

import FullScreen from '../layouts/FullScreen';

export const Home: NextPage = () => {
  const { t } = useTranslation('common');
  return (
    <FullScreen>
      <main className="flex align--center justify-center flex-column">
        <h1 className="flex">{t('main.title')}</h1>
        <Button>
          <Link href="/signin">{t('home.signin')}</Link>
        </Button>
      </main>
    </FullScreen>
  );
};

export default Home;
