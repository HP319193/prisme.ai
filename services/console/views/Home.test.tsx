import Home from './Home';
import renderer from 'react-test-renderer';

jest.mock('next/router', () => {
  const mock = {
    replace: () => null,
  };
  return {
    useRouter: () => mock,
  };
});

it('should render', () => {
  const root = renderer.create(<Home />);
  expect(root.toJSON()).toMatchSnapshot();
});
