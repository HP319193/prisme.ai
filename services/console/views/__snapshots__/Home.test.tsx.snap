// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="flex justify-content-center align-items-center min-h-screen"
>
  <main
    className="flex align-items-center justify-content-center flex-column"
  >
    <h1
      className="flex"
    >
      home.title
    </h1>
    <button
      className="p-button p-component"
      disabled={false}
    >
      <a
        href="/signin"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        home.signin
      </a>
    </button>
  </main>
</div>
`;
